ui/apps/platform/src/Components/CompoundSearchFilter/types.ts

```diff
@@ -323,10 +323,10 @@ export const nodeCVESearchFilterConfig = {
     displayName: 'Node CVE',
     searchCategory: 'NODE_VULNERABILITIES',
     attributes: {
-        ID: {
-            displayName: 'ID',
-            filterChipLabel: 'Node CVE ID',
-            searchTerm: 'CVE ID',
+        Name: {
```

Comment:
This change in Components folder and deletions in Containers/Vulnerabilities folder makes me hope we can refactor container-specific advanced filter types during 4.6A sprint.

Reply:
For your reference, I am [re-exporting these types from a file](https://github.com/stackrox/stackrox/pull/11333/files#r1631606782) underneath `Containers/Vulnerabilities` and using that as the source of imports for everything in VM. I'm hoping this leads to minimal changes in 4.6A where we can inline the values that are currently under the `Component/` level in this top level file without creating a diff at every import point.